Description: Remove -msse2 build option for gcc.
 This flag is only available for amd64 and i386 archs.
Forwarded: yes
Bug-Debian: http://bugs.debian.org/479809
Last-Update: 2011-03-10
---
 include/STTypes.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- soundtouch.orig/include/STTypes.h
+++ soundtouch/include/STTypes.h
@@ -93,7 +93,7 @@ namespace soundtouch
         /// routines compiled for whatever reason, you may disable these optimizations 
         /// to make the library compile.
 
-        #define SOUNDTOUCH_ALLOW_X86_OPTIMIZATIONS     1
+        // #define SOUNDTOUCH_ALLOW_X86_OPTIMIZATIONS     1
 
         /// In GNU environment, allow the user to override this setting by
         /// giving the following switch to the configure script:
